### PR TITLE
[YPPY-1615] - Upgrade deps

### PR DESCRIPTION
**Pull request by FalconerTC**
Issue: [YPPY-1615](https://renttherunway.jira.com/browse/YPPY-1615)
Various upgrades:
 - Upgrade amqp-client to 5.x
 - Remove powermock
 - Update license
 - Add Jenkinsfile

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,96 +9,23 @@
...
-        <developer>
-            <name>Carlo Barbara</name>
-            <email>carlo.barbara@gmail.com</email>
-            <timezone>America/New_York</timezone>
-            <roles>
-                <role>committer</role>
-            </roles>
-        </developer>
-        <developer>
-            <name>Gene Trog</name>
```
Comment (gtrog) [inline pom.xml:35]: 😢

File: src/test/java/io/rtr/conduit/AMQPIntegrationIT.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+            // Sleep for consistency
+            sleep(500);
+        } catch (IOException | TimeoutException | InterruptedException e) {
+            // gotta catch 'em all!
+            throw new AssertionError("Publishing threw an exception", e);
+        }
+    }
+
+    private Consumer buildConsumer() {
```
Comment (gtrog) [inline src/test/java/io/rtr/conduit/AMQPIntegrationIT.java:82]: `static`

File: src/test/java/io/rtr/conduit/AMQPIntegrationIT.java [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                        EXCHANGE,
+                        AMQPConsumerBuilder.ExchangeType.DIRECT,
+                        QUEUE,
+                        ROUTING_KEY
+                )
+                .callback(callback)
+                .build();
+    }
+
+    private Publisher buildPublisher() {
```
Comment (gtrog) [inline src/test/java/io/rtr/conduit/AMQPIntegrationIT.java:101]: `static`

File: pom.xml (hunk 1 above)
Comment (FalconerTC) [reply pom.xml:35]: No idea who this "Gene Trog" fellow was

Comment (gtrog) [reply pom.xml:35]: Good point hehe

**Comment by FalconerTC** [comment]
Build failure is because of the added Jenkinsfile not being in master

**Pull request merged**